guides/common/modules/proc_integrating-a-generic-dns-server-by-using-dns-nsupdate.adoc

```diff
@@ -49,6 +52,8 @@ send\n" | nsupdate -k /etc/foreman-proxy/rndc.key
 --foreman-proxy-dns-server="_DNS_IP_Address_" \
 --foreman-proxy-keyfile=/etc/foreman-proxy/rndc.key
 ----
+
 . In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
```

Comment:
This was part of the dropped Refresh step so can be dropped too
```suggestion
```

Reply:
removed